src/ui.c

```diff
@@ -2,6 +2,7 @@
 #include "ui.h"
 
 #include "msemu.h"
+#include "config.h"
```

Comment:
Isn't it best practice to have config.h be the first include?

Reply:
I honestly wasn't sure, but that makes sense to me.